src/styles/_PopUp.scss

```diff
@@ -61,7 +62,7 @@
   margin: 10px auto;
   width: 90%;
   padding-bottom: 15px;
-
+  text-transform: capitalize;
```

Comment:
Makes all of the simple/average/complex difficulty ratings capitalized on each game card